### ISSUE TITLE
make FilePicker double as a notepad 

### ISSUE THREAD

**Issue by newton1985** [ITISFoundation/osparc-issues#1195]
**Describe the user role**
osparc/sim4life-web user creating complex pipelines 

**Describe the goal**
I would like a convenient way of inspecting and editing text files that I drag and drop onto the workbench (or upload via a filepicker) without editing the file locally, then re-uploading.

**Describe the benefit**
This would be extremely convenient and save quite a bit of time, as pipelines frequently need to be tested by altering small inputs such as text files.

**Additional context**

**Comment by elisabettai** [comment]
Thank you @newton1985 for creating the issue. 
Just to understand better the use-case and how urgent this is: which service will consume this text file? Obviously, if the consumer is a jupyterlab there's a text editor already integrated (jupyter-smash has even more advanced text editors, such as vscode).

If it's a computational service, then there the only way to edit the file for the moment is to do it locally outside oSPARC.

**Comment by newton1985** [comment]
@elisabettai that's true - but I thought that jupyterlab NB input ports are read-only?

**Comment by elisabettai** [comment]
Hi @newton1985, yes, it is true: input ports in jupyterlab are read-only. Bringing the input file to the workspace and edit it when there is an option, maybe not ideal.
Yes, as you said if this "file picker text editor" would exist, it will work the same for the downstream services, who would get the updated file in the input as soon as there's a change.
The reason I was asking if it was computational is that in this case having a "file picker text editor" would be more critical, as computational services don't have the editing functionality of jupyterlabs,

**Comment by newton1985** [comment]
@elisabettai - ok, sorry, yes I see your point now! It's definitely not an immediate emergency - just a feature that would make the UX nicer imo :).
